tests/common/fixtures/conn_graph_facts.py

```diff
@@ -15,7 +15,7 @@ def conn_graph_facts(duthost, localhost):
         inv_opt = duthost.host.options['inventory']
         inv_files = []
         if isinstance(inv_opt, str):
-            inv_files = (duthost.host.options['inventory'])  # Make it iterable for later use
+            inv_files = [duthost.host.options['inventory']]  # Make it iterable for later use
```

Comment:
is this a comma separated list? Looks like this [line](https://github.com/Azure/sonic-mgmt/blob/master/tests/common/fixtures/advanced_reboot.py#L124) is broken too

Reply:
It is a comma separated list. refer #1645. @tahmed-dev , can you fix that line as a separate PR. Not sure if there are other places as well